### DISCUSSION TITLE
Mounting points

### DISCUSSION THREAD

**Discussion by Oceloot** [FundamentalFrequency/.github#14] [MCAD/ECAD]
_**(hope this was the right category for this)**_ 
hey, saw that there are mounting holes on the pcb but from my understanding they arent being used in the lasercut case? so the only place the pcb is mounted to the case is through the switches? does the pcb flex when you for an example press down on the encoder to the right or press down on the "joystick"? if i were to design my own case i guess it would be best to use the mounting holes to mount the pcb to the bottom of the case with standoffs or something? ive modified the pcb files to take low profile switches (im a noob at this though so the traces are a bit all over the place). and the low profile switches wont be able to be mounted in a 3mm plate so i guess ill just use the mounting points and then design my own case. that would be the best right?

**Comment by stonepreston** [comment]
Yes the only place the PCB is mounted currently is via the soldered switches. There a small amount of flex when you push on the joystick (although the joystick button does not do anything currently, so I dont push on it), but not on any of the encoders.
